github_connector/models/github_repository_branch.py

```diff
@@ -227,6 +227,13 @@ def _get_analyzable_files(self, existing_folder):
         return res
 
     def set_analysis_rule_info(self):
+        """Remove all analysis info previously + rule process.
```

Comment:
This is not a docstring, but a comment about the following line. Dosctrings is for documenting what the method does, which is not the case.